src/world/server.rs

```diff
@@ -323,6 +323,29 @@ pub async fn server_main_loop(mut recv: Receiver<ToServer>) -> Result<(), std::i
                                 to_remove.push(id);
                             }
                         }
+
+                        if let ClientTriggerCommand::EventRelatedUnk {
+                            unk1: _,
+                            unk2: _,
+                            unk3: _,
+                            unk4: _,
+                        } = &trigger.trigger
```

Comment:
```suggestion
                        if let ClientTriggerCommand::EventRelatedUnk { .. } = &trigger.trigger
```

Is a much shorter way to write "i don't care about any values of this struct"